### PR TITLE
Fix: Broken link of website and discord

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Type of Change
- [X] New feature (non-breaking change which adds functionality)
- [ ] Fixes Bug (Fixes an existing functionality which was not working as expected)
- [ ] Documentation change (Changes to Readme.md)

## Describe your changes
[A brief overview of the feature being added. Explain the functionality and its intended purpose.]
Fix: Broken link of website and discord in README.md
## Screenshots/Videos (if applicable)
[Attach screenshots or videos demonstrating the new feature in action.]
Loom Video: [loom](https://www.loom.com/)

## Checklist
- [x] I have taken a (git pull origin main --rebase) from main branch and tested it.
- [ ] I have conducted a self-review of my code, and the code contains only changes relevant to this feature, without including modifications to other files.
- [ ] I have commented my code, particularly in complex areas.
- [x] I have checked for compatibility with other parts of the codebase by manually running the site and doing actions not related to my changes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
     <a href="https://www.vidurjobs.xyz"><strong>Learn more »</strong></a>
```
Comment (amandesai01) [inline docs/README.md:15]: change this too

File: docs/README.md [hunk 2]
```diff
@@ -15,9 +15,9 @@
...
     <br />
     <br />
-    <a href="https://discord.gg/Pt2W9Sx7ap">Discord</a>
+    <a href="https://discord.com/invite/cRaukv9dJ2">Discord</a>
```
Comment (amandesai01) [inline docs/README.md:18]: revert this. let @devshahofficial add the official link

**Review by amandesai01 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (shivam-sharma7) [reply docs/README.md:15]: @amandesai01 okay but I need link for this change

File: docs/README.md (hunk 2 above)
Comment (shivam-sharma7) [reply docs/README.md:18]: @amandesai01 okay, reverting this

**Review by amandesai01 — APPROVED** [review]

**Pull request merged**